### ISSUE TITLE
enableSync and arrayContentWillChange/arrayContentDidChange causes rows to duplicate

### ISSUE THREAD

**Issue by rhyek** [offirgolan/ember-light-table#196]
So I'm testing out using `enableSync` and sorting my array with `Array.prototype.sort` and using `arrayContentWillChange/DidChange`, but for some reason when doing so each row is duplicated every time I sort.

You can have a look at what I'm doing if you clone this repo: https://github.com/rhyek/ember-light-table-test.

Thanks.

Edit:
The relevant code is here: https://github.com/rhyek/ember-light-table-test/blob/master/app/components/my-light-table.js

**Comment by alexander-alvarez** [comment]
I came across this too when trying to replace the rows, with `enableSync` being true. I had to clear the rows before setting them

```js
this.get('table').setRows();
this.get('table').setRows(sortedRows);
```

**Comment by offirgolan** [comment]
Hmmm okay, I'll take a look into this!

**Comment by offirgolan** [comment]
Released the fix in `v1.4.3` 😺
